src/index.js

```diff
@@ -3,29 +3,25 @@ import axios from "axios";
 import fs from "fs";
 import shell from "shelljs";
 
+function commits_for(contribution) {
+  return `GIT_AUTHOR_DATE=${contribution.date}T12:00:00 GIT_COMMITER_DATE=${contribution.date}T12:00:00 git commit --allow-empty -m "Rewriting History!" > /dev/null\n`.repeat(contribution.count)
+}
+
 export default async (input) => {
   const res = await axios.get(
     `https://github.com/users/${input.username}/contributions?tab=overview&from=${input.year}-12-01&to=${input.year}-12-31`
   );
 
   // Gathers all the squares from GitHub contribution graph.
-  const allSquares = parse(res.data).querySelectorAll("[data-count]");
-
-  const script = allSquares
-    .reduce((fullCommand, contribution) => {
-      const count = contribution._rawAttrs["data-count"];
-      const date = contribution._rawAttrs["data-date"];
-
-      // Only care about html elements of squares with 1+ contributions for that day.
-      if (count > 0) {
-        return (fullCommand +=
-          `GIT_AUTHOR_DATE=${date}T12:00:00 GIT_COMMITER_DATE=${date}T12:00:00 git commit --allow-empty -m "Rewriting History!" > /dev/null\n`.repeat(
-            count
-          ));
-      }
-      return fullCommand;
-    }, "#!/bin/bash \n")
```

Comment:
The shebang is unnecessary because the file is being explicitly executed by `bash script.sh`. If we were expecting to keep the script around, using the shebang and `chmod` the file to make it executable could be an option